api/src/main/java/org/aya/api/ref/DefVar.java

```diff
@@ -54,4 +54,11 @@ private DefVar(Concrete concrete, Core core, @NotNull String name) {
   @Override public boolean equals(Object o) {
     return this == o;
   }
+
+  public boolean isInModule(@NotNull ImmutableSeq<String> moduleName) {
+    var maybeSubmodule = module;
```

Comment:
Does inlining this variable make any sense?

Reply:
Just remind people that we should check submodules instead of simple `currentModule.equals(module)`